modules/js/localStorage.js

```diff
@@ -0,0 +1,8 @@
+export function saveToLocalStorage(key, data) {
+  localStorage.setItem(key, JSON.stringify(data));
+}
```

Comment:
- [ ] Consider using arrow functions instead of the`function` keyword to leverage ES6 features in JavaScript. Arrow functions offer concise syntax and prevent potential scoping bugs. Please give it a try to enhance code readability and maintainability.

Reply:
- [ ] As mentioned by the previous reviewer, kindly change this function to an arrow function to follow the project requirement (see image)
![Screenshot from 2023-07-19 07-46-27](https://github.com/mumbereh/Awesome-Book/assets/91142432/9574c300-84c0-4ded-be7c-e8ccedaa0d0a)


> * [ ]  Consider using arrow functions instead of the`function` keyword to leverage ES6 features in JavaScript. Arrow functions offer concise syntax and prevent potential scoping bugs. Please give it a try to enhance code readability and maintainability.